image/tarexport/tarexport.go

```diff
@@ -30,18 +32,36 @@ type tarexporter struct {
 	loggerImgEvent LogImageEvent
 }
 
+type ocitarexporter struct {
+	te   *tarexporter
+	name string
```

Comment:
I have a vague memory that they were different in a working version but that didn't pass the review. Imho they can be different objects in implementation but they should live in the same package(note the package name also has `export` in the name atm). We can also just find a better name for the var.